katz/src/main/kotlin/katz/instances/IOSemigroup.kt

```diff
@@ -0,0 +1,11 @@
+package katz
+
+class IOSemigroup<A>(val SM: Semigroup<A>) : Semigroup<HK<IO.F, A>> {
```

Comment:
should this use default params for the registered instance value if not provided at instantiation?